utils/land-patterns.stanza

```diff
@@ -151,6 +151,34 @@ public defn rect-pth-pad (drill-r:Double, pad-w:Double, pad-h:Double) :
 public defn rect-pth-pad (drill-w:Double, drill-h:Double, pad-w:Double, pad-h:Double) :
   rect-pth-pad(C, drill-w, drill-h, pad-w, pad-h)
 
+pcb-pad roundrect-pth-pad (anchor:Anchor, drill-w:Double, drill-h:Double, pad-w:Double, pad-h:Double, pad-r:Double) :
+  name = to-string("%_ RoundedRectangle PTH Pad (drill = (%_, %_), pad = (%_, %_, %_))" % [
+           anchor, drill-w, drill-h, pad-w, pad-h, pad-r])
+  type = TH
+  if drill-w != drill-h :
```

Comment:
if these must be equal, we should have one `drill-r` argument instead of two, I think

Reply:
Comment relocated: If `drill-w != drill-h` then the cutout layer can be a capsule. `rect-pth-pad` and `oval-pth-pad` demonstrate that.